core/src/main/scala/sttp/client3/internal/ToRfc2616Converter.scala

```diff
@@ -0,0 +1,63 @@
+package sttp.client3.internal
+
+import sttp.client3._
+import sttp.model._
+
+class ToRfc2616Converter[R <: RequestT[Identity, _, _]] {
+  def apply(request: R): String = {
+    val extractMethod = request.method.method
+    val extractUri = request.uri
+    val result = s"$extractMethod $extractUri"
+    val headers = extractHeaders(request)
+    val resultWithHeaders = if (headers.isEmpty) result else result + s"\n$headers"
+    val body = extractBody(request)
+    if (body.isEmpty) resultWithHeaders else resultWithHeaders + s"\n\n$body"
+  }
+
+  private def extractBody(r: R): String = {
+    r.body match {
+      case StringBody(text, _, _) => s"$text"
+      case ByteArrayBody(_, _)    => "<PLACEHOLDER>"
```

Comment:
so these are effectively not supported?

Reply:
I can't find any examples in documentation, so not. I also asked in issue about that cases - whether we need them.